lib/_stream_readable.js

```diff
@@ -1081,6 +1081,13 @@ Object.defineProperty(Readable.prototype, 'readableLength', {
   }
 });
 
+Object.defineProperty(Readable.prototype, 'readableObjectMode', {
+  enumerable: false,
+  get() {
+    return this._readableState && this._readableState.objectMode;
```

Comment:
Maybe return `false` instead of `undefined` if `this._readableState === undefined` when the stream wasn't instantiated ?

Refs: https://github.com/nodejs/node/pull/27190#discussion_r275705847